config/i18n.json

```diff
@@ -467,6 +469,8 @@
     "SaveAs": "Save as",
     "SeasonNum": "Season {{season}}",
     "SelectMax": "- Max -",
+    "SelectMaxEnergy": "Maximum Energy",
```

Comment:
Without seeing the title popup I was a little confused to what this meant. The title helps but a lot of users might not hover long enough to see it. How do you feel about "Ignore Element Affinity" and "Ignored Energy Threshold". It's still shorter than our other config titles.

I'm not  really great with this stuff though, so if you have a better idea go for it.

![image](https://user-images.githubusercontent.com/7344652/101224913-22d8a980-36e4-11eb-95b1-764887778e07.png)